pkg/gadgets/run/tracer/tracer.go

```diff
@@ -453,12 +453,12 @@ func (t *Tracer) processEventFunc(gadgetCtx gadgets.GadgetContext) func(data []b
 		}
 
 		return &types.Event{
-			Type:          eventtypes.NORMAL,
-			WithMountNsID: eventtypes.WithMountNsID{MountNsID: mtn_ns_id},
-			RawData:       data,
-			L3Endpoints:   l3endpoints,
-			L4Endpoints:   l4endpoints,
-			Timestamps:    timestamps,
+			Type:        eventtypes.NORMAL,
+			MountNsID:   mntNsId,
```

Comment:
Wouldn't we need to fill out the `NetNsID` here too? (We didn't do it before your PR too so we can fix it in another one)
AFAIK that field is only filled for snapshotters in https://github.com/inspektor-gadget/inspektor-gadget/blob/cbe1bb88d4d818396b1d78da23e91bdc15e327c5/pkg/gadgets/run/tracer/tracer.go#L512C33

Reply:
I think it makes sense. We'll have to define a `gadget_netns_id` type as done for the mount namespace and understand the implications on the enrichment logic -> https://github.com/inspektor-gadget/inspektor-gadget/issues/1548